docs/intro/tutorial.rst

```diff
@@ -490,30 +551,109 @@ Another common pattern is to build an item with data from more than one page,
 using a :ref:`trick to pass additional data to the callbacks
 <topics-request-response-ref-request-callback-arguments>`.
 
+Another example: scraping authors
```

Comment:
What do you think about moving the section above (a link to the trick to pass additional data to callbacks) below this example?